### DISCUSSION TITLE
Add Client-side Searching via Fuse.js to Static Sites | /en/2023/09/fuse-search/

### DISCUSSION THREAD

**Discussion by giscus** [rbind/yihui#1512] [Comments]
Normally web searching is performed on the server side. That is, when you send a query with some keywords, the server will search its database that contains all data of a website. Searching can also &hellip;

https://yihui.org/en/2023/09/fuse-search/

**Comment by gepcel** [comment]
博客内容中，Markdown的换行，在转换成html源代码中得以保留，并在html页面中正确地转换为一个空格。但在某feed客户端下，空格和换行都消失，导致单词连在一起。倒是没注意是刚出现的还是以前一直存在。例如：

![image](https://github.com/gepcel/GisViewer/assets/4179106/fac2bff0-4958-44eb-99f7-6d10c7206627)

**Reply by yihui** [reply]
谢谢反馈。我也注意到了，但也不知道是 Feedly 最近出了什么幺蛾子，还是怎么回事。感觉 Feedly 对 RSS 中读取的 HTML 代码没太处理好。我想了两个小时，用了一个暗黑的办法把换行符替换为空格了：https://github.com/yihui/yihui.org/commit/a7a2eca855bd668d87975a40783d47b3b3a451a2

我还是建议到网站上来读文章，排版会好一点，而且有时候有些例子在 RSS 阅读器中无法运行。

**Reply by gepcel** [reply]
基本上都是会到网站读文章的。网站的排版设计很精美。feedly只是用来看到有更新了，今天无意中注意到的这个。
